lib/Models/Catalog/Gtfs/prettyPrintGtfsEntityField.ts

```diff
@@ -9,15 +9,15 @@ export default function prettyPrintGtfsEntityField(
     // TODO: Get sone of this data (eg. route short name) from static GTFS csv files instead
     // This probably only works for NSW
     case "route_short_name": {
-      const route: string = _get(entity, "vehicle.trip.route_id");
+      const route: string = _get(entity, "vehicle.trip.route_id", "");
       if (route !== undefined && route.indexOf("_") + 1 > 0) {
```

Comment:
I think because it is checking for `route !== undefined` - the safer change would be to type it as `route: string | undefined`  (similar to what you have done below) - instead of passing a default empty string.

Reply:
I'm pretty sure I tried that first and it didn't work, but let's go with that for now. Fixed.